downstream/modules/platform/proc-controller-inv-source-vm-esxi.adoc

```diff
@@ -0,0 +1,29 @@
+[id="proc-controller-inv-source-vm-esxi"]
+
+= VMware ESXi
+
+Use the following procedure to configure a VMWare-ESXI sourced inventory.
+
+.Procedure
+. From the navigation panel, select {MenuInfrastructureInventories}.
+. Select the inventory name you want a source to and click the *Sources* tab.
+. Click btn:[Create source].
+. In the *Create source* page, select *VMware ESXi* from the *Source* list.
+. The *Create source* window expands with additional fields.
+Enter the following details:
+
+* Optional: *Credential*: Choose from an existing VMware credential.
+For more information, see xref:controller-credentials[Managing user credentials].
+
+. Optional: You can specify the verbosity, host filter, enabled variables or values, and update options as described in xref:proc-controller-add-source[Adding a source].
```

Comment:
The verbosity appears to be required too.

Reply:
![image](https://github.com/user-attachments/assets/4777d027-7a62-4c6b-bfc7-85e6d9d9e2ff)